packages/js-sdk/src/sandbox/sandboxApi.ts

```diff
@@ -132,7 +142,7 @@ export class SandboxApi {
     }
 
     return (
-      res.data?.map((sandbox: components['schemas']['RunningSandbox']) => ({
+      res.data?.map((sandbox: components['schemas']['ListedSandbox']) => ({
```

Comment:
the expected return type is `SandboxInfo`, that's not the same, can you also split